### ISSUE TITLE
discord-netflix and Linux

### ISSUE THREAD

**Issue by MeixDev** [nirewen/discord-netflix#17]
Hi !

I wasn't able to get discord-netflix to work on my Arch Linux.
To be right, it works as expected as a Netflix wrapper, but it doesn't support Rich Presence at all.

The README.md talks about an error showing up for 15 seconds if Discord is closed when launching discord-netflix, but it doesn't appear.
We could add that it's not automatically recognized by Discord as a game, but that may be normal.

Have a good day !

**Comment by Wist9063** [comment]
Were there any errors in the terminal?

**Comment by nirewen** [comment]
Can you test if it will work now? 
It's supposed to work on Linux, I think it will download electron-widevinecdm 7.0.0, if not, let me know.

**Comment by MeixDev** [comment]
Great news : It now works !

I had a WARN from npm after npm install telling me that I had to install peer dependencies myself.
I executed "npm i register-scheme@github:devsnek/node-register-scheme" then tried npm start, and it launched as expected.

I didn't have time to test it more (I will do it later, if needed), but Discord recognized Discord Netflix immediately, and achieved to display my icon, and the fact that I was browsing the catalog. Great improvement !

I just checked, but it apparently downloaded electron-widevinecdm 5.0.2, and not the 7.0.0. It still worked, so I won't complain.

Good job, everyone (: :+1:
